spn/index.md

```diff
@@ -3,4 +3,6 @@ title: SPN
 layout: base
 ---
 
-SPN Base Page
+The expanded SPN documentation is in progress.
+
+You can find more information about the SPN [on our website]({{ site.spn_url }})
```

Comment:
maybe something like "will be added in the future"

"in progress" quickly becomes a lie when priorities shift